.github/workflows/win_cpn-64.yml

```diff
@@ -1,4 +1,4 @@
-name: Windows Companion 64-bit
```

Comment:
You should not have any changes in this file: remove them.

Reply:
I'm also confused. I didn't change them manually, like they were changed during compilation. Fortunately, I have restored them.